### ISSUE TITLE
완료 했습니다.

### ISSUE THREAD

**Issue by cryscham123** [cryscham123/minishell#22]
월요일날에 클러스터 가서 더 점검해볼 예정인데 일단은 돌아가게 만들어 놨습니다.

**Comment by cryscham123** [comment]
시그널 처리하는 부분 조금 수정했는데, minishell 안에서 minishell 실행할 때, 출력이 이상하게 되서 자식 프로세스를 끝내고 받은 flag 값을 바탕으로 부모 프로세스에서 메세지 (SIG_INT: ^C\n, SIG_QUIT: ^\QUIT:3\n) 출력해주는 식으로 변경했습니다.

**Comment by cryscham123** [comment]
매크로 함수 안쓰려고 했는데 minishell 안에서 minishell 실행하고 exit (number) 이런식으로 명령어를 사용했을 때 exit status가 제대로 적용이 안되더라고요. 그래서 매크로 함수를 사용해야 할것 같습니다.

**Comment by lucku-1111** [comment]
네 알겠습니다!

**Comment by cryscham123** [comment]
보너스 부분에 "'$PATH'" 이 부분도 사실상 맞추는게 맞는것 같아서 작동하도록 수정했습니다. 대신에 resplit 처리하는게 조금 그래서 걍 없앴습니다. 그리고 exit status는 이제 126번 리턴값 ( 실행 가능한 파일이 있는데 접근 권한이 없을 때 리턴값) 이랑 2번 리턴값 ( built in 함수 이상하게 사용했을 때) 제외하고는 다 bash랑 동일하게 작동할것 같아요. 126번은 구조를 다시 짜야되서 걍 안하는게 맞는거 같네요

**Comment by lucku-1111** [comment]
ambiguous redirect 에러가 처리가 안되는데 방식이 바뀌었나요??

**Comment by cryscham123** [comment]
네 resplit 하는 부분 완전히 없앴어요

**Comment by lucku-1111** [comment]
아아 네 알겠습니다! 내일 강남으로 가시나요?

**Comment by cryscham123** [comment]
네. resplit은 원래 구현했던것도 이상했었고 (echo > "a  b c   d" )이런 경우나 (export yo="    a b   c d     "; echo > $yo) 사이가 아니라 앞 뒤에 공백이 있는 경우, 구현 바꾸다보니까 꼬이는것 같아서 그냥 없앴습니다. 그리고 EOF 부분 조금 이상하게 동작하는것 같아서 수정해서 다시 push 했어요.

**Comment by lucku-1111** [comment]
네 알겠습니다! 내일 클러스터에서 이것저것 확인해보죠!

**Comment by cryscham123** [comment]
개포에서 마무리해서 저녁에 평가 하나만 받아보는건 어떤가요

**Comment by lucku-1111** [comment]
그것도 좋습니다!
